domain/src/Services/TransactionDispatcher/Handlers/TransferHandler.php

```diff
@@ -30,17 +30,17 @@ public function handle(Transaction $transaction): void
         $taxYearId = TaxYearId::fromTaxYear($taxYear);
         $taxYearAggregate = $this->taxYearRepository->get($taxYearId);
 
-        if ($transaction->transactionFeeCostBasis?->isGreaterThan('0')) {
+        if ($transaction->networkFeeMarketValue?->isGreaterThan('0')) {
             $taxYearAggregate->recordNonAttributableAllowableCost(new RecordNonAttributableAllowableCost(
                 taxYear: $taxYear,
-                amount: $transaction->transactionFeeCostBasis,
+                amount: $transaction->networkFeeMarketValue,
             ));
         }
 
-        if ($transaction->exchangeFeeCostBasis?->isGreaterThan('0')) {
+        if ($transaction->platformFeeMarketValue?->isGreaterThan('0')) {
             $taxYearAggregate->recordNonAttributableAllowableCost(new RecordNonAttributableAllowableCost(
                 taxYear: $taxYear,
-                amount: $transaction->exchangeFeeCostBasis,
+                amount: $transaction->platformFeeMarketValue,
             ));
         }
```

Comment:
ℹ️ Transaction fees and exchange fees have been renamed for network fees and platform fees respectively